.github/workflows/release.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         run: twine check --strict dist/*
 
       - name: Generate artifact attestation for sdist and wheel
-        uses: actions/attest-build-provenance@173725a1209d09b31f9d30a3890cf2757ebbff0d # v1.1.2
+        uses: actions/attest-build-provenance@210c1913531870065f03ce1f9440dd87bc0938cd # v1.4.0
```

Comment:
We should also verify if this breaks anything before merging.

Reply:
Hmm, we don't really have a way to dry run the release action right?

Reply:
I will test this before merging (via a test release). I have a grant due Thursday, so I won't get to this before Friday.